### PR TITLE
Belarusian Cheat Sheet: Add transliteration

### LINKED ISSUE #3260: Belarusian Cheat Sheet: Should show pronunciation
For language cheat sheets we have a `trn` property (as well as `key` and `val`) for transliteration/pronunciation. The Belarusian cheat sheet should be updated to include them. See the Chinese cheat sheet for a live example: https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/language/chinese.json   

[Suggested on Twitter](https://twitter.com/NiklasNisbeth/status/743435020803653632)

------
IA Page:  http://duck.co/ia/view/belarusian_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @qqq3

### PR DESCRIPTION
**Pull request by qqq3**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
Add transliteration

###### Which issues (if any) does this fix?

Fixes #3260 

---

Instant Answer Page: https://duck.co/ia/view/belarusian_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
I think "transliteration is one of my favorite words.  Thanks for another PR @qqq3

**Comment by tagawa** [comment]
Hi @qqq3. Thanks for this - I've just installed it on the beta server for testing: https://beta.duckduckgo.com/?q=Belarusian+Cheat+Sheet&ia=cheatsheet&iax=1

I'll ask for some language expert verification just to make sure it's OK.

**Comment by tagawa** [comment]
@mialeska @denisbalyko Would you have time to check the transliteration that's been added to these phrases please?

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 1]
```diff
@@ -23,224 +23,277 @@
...
             },
             {
                 "key": "Welcome!",
+                "trn": "Sardjechna zaprashaem!",
                 "val": "Сардэчна запрашаем!"
             },
             {
                 "key": "How are you?",
+                "trn": "Jak maeshsja?",
                 "val": "Як маешся?"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Як ты? || Як справы?

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 2]
```diff
@@ -23,224 +23,277 @@
...
             },
             {
                 "key": "Taxi!",
+                "trn": "Taksi!",
                 "val": "Таксі!"
             },
             {
                 "key": "Hotel",
+                "trn": "Gatjel'",
                 "val": "Гатэль"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Атэль (possibly)

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 3]
```diff
@@ -23,224 +23,277 @@
...
             },
             {
                 "key": "Dessert",
+                "trn": "Djesert",
                 "val": "Дэсерт"
             },
             {
                 "key": "The check please",
+                "trn": "Chjek, kalі laska",
                 "val": "Чэк, калі ласка"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Рахунак, калі ласка

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 4]
```diff
@@ -23,224 +23,277 @@
         "Basics": [
             {
                 "key": "Hello",
+                "trn": "Vitayu",
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Vitaju

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 5]
```diff
@@ -23,224 +23,277 @@
...
             },
             {
                 "key": "What is your name?",
+                "trn": "Jak vas klіchuc'?",
                 "val": "Як вас клічуць?"
             },
             {
                 "key": "My name is ...",
+                "trn": "Mjane zvac' ...",
                 "val": "Мяне зваць ..."
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Мяне клічуць ...

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 6]
```diff
@@ -23,224 +23,277 @@
...
         "Getting help": [
             {
                 "key": "Can you help me?",
+                "trn": "Cі mozhace vy mne dapamagchy?",
                 "val": "Ці можаце вы мне дапамагчы?"
             },
             {
                 "key": "Where is the toilet?",
+                "trn": "Dze znahodzіca tualet?",
                 "val": "Дзе знаходзіца туалет?"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Дзе знаходзіцца прыбіральня

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 7]
```diff
@@ -23,224 +23,277 @@
...
             },
             {
                 "key": "Straight ahead",
+                "trn": "Prama",
                 "val": "Прама"
             },
             {
                 "key": "Where is the exit?",
+                "trn": "Dze vyhad?",
                 "val": "Дзе выхад?"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Дзе выйсце?

**Comment by denisbalyko** [comment]
I'm not good with translit (by to lat), sorry.

**Comment by tagawa** [comment]
No problem @denisbalyko - thanks for the comments! It looks like most of them are for the original translations rather than transliterations. @qqq3 - do they look OK to you?

File: share/goodie/cheat_sheets/json/language/belarusian.json (hunk 5 above)
Comment (qqq3) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3109#discussion-diff-65886310

File: share/goodie/cheat_sheets/json/language/belarusian.json (hunk 1 above)
Comment (qqq3) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3109#discussion-diff-65886173

**Comment by tagawa** [comment]
@qqq3 To finish this off, why don't we leave the original translations and just make the single change to the transliterations - line 26, "Vitayu" -> "Vitaju".

Then we could make a separate issue to look into translation edits if that's necessary.

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 8]
```diff
@@ -23,224 +23,277 @@
         "Basics": [
             {
                 "key": "Hello",
+                "trn": "Vitayu",
                 "val": "Вітаю"
             },
             {
                 "key": "Welcome!",
+                "trn": "Sardjechna zaprashaem!",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Sorry, but this transliteration looks weird, especially for Э and Е belarusian letters It's better to use tranliterations mentioned there:
https://en.wikipedia.org/wiki/Romanization_of_Belarusian
If you not afraid of using special symbols, I would like to ask you to use this one below, it's the best one from the point of view of belarusian linguistic and also modern. In Belarus you can see it on the street navigation signs, in the metro and also on maps. It's based on classic belarusian latin alphabet, and has official status for some geogr. purposes:
https://en.wikipedia.org/wiki/Instruction_on_transliteration_of_Belarusian_geographical_names_with_letters_of_Latin_script
Also you can use online romanizer for your purposes, this one is really good: http://baltoslav.eu/lat/

For this line I suggest you "Sardečna zaprašajem" or at least "Sardechna zaprashajem"
@tagawa

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 9]
```diff
@@ -23,224 +23,277 @@
...
         ],
         "Days of the Week": [
             {
                 "key": "Monday",
+                "trn": "Panjadzelak",
                 "val": "Панядзелак"
             },
             {
                 "key": "Tuesday",
+                "trn": "Aўtorak",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Aŭtorak

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 10]
```diff
@@ -23,224 +23,277 @@
...
             {
                 "key": "The check please",
+                "trn": "Chjek, kalі laska",
                 "val": "Чэк, калі ласка"
             }
         ],
         "Days of the Week": [
             {
                 "key": "Monday",
+                "trn": "Panjadzelak",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Paniadzielak

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 11]
```diff
@@ -23,224 +23,277 @@
...
                 "val": "Серада"
             },
             {
                 "key": "Thursday",
+                "trn": "Chacver",
                 "val": "Чацвер"
             },
             {
                 "key": "Friday",
+                "trn": "Pjatnіca",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Piatnica

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 12]
```diff
@@ -23,224 +23,277 @@
...
                 "val": "Панядзелак"
             },
             {
                 "key": "Tuesday",
+                "trn": "Aўtorak",
                 "val": "Аўторак"
             },
             {
                 "key": "Wednesday",
+                "trn": "Serada",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Sierada

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 13]
```diff
@@ -23,224 +23,277 @@
...
                 "val": "Серада"
             },
             {
                 "key": "Thursday",
+                "trn": "Chacver",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Čacvier or Chacvier

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 14]
```diff
@@ -23,224 +23,277 @@
...
                 "val": "Пятніца"
             },
             {
                 "key": "Saturday",
+                "trn": "Subota",
                 "val": "Субота"
             },
             {
                 "key": "Sunday",
+                "trn": "Njadzelja",
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Niadzielia

File: share/goodie/cheat_sheets/json/language/belarusian.json (hunk 8 above)
Comment (edgesince84) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: thanks a ton for the help! @mialeska

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 15]
```diff
@@ -23,224 +23,277 @@
...
             {
                 "key": "Taxi!",
+                "trn": "Taksi!",
                 "val": "Таксі!"
             },
             {
                 "key": "Hotel",
-                "val": "Гатэль"
+                "trn": "Gatjel'",
+                "val": "Aтэль"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: I think the `trn` needs to change to match the new `val`

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 16]
```diff
@@ -23,224 +23,277 @@
...
             {
                 "key": "Straight ahead",
+                "trn": "Prama",
                 "val": "Прама"
             },
             {
                 "key": "Where is the exit?",
-                "val": "Дзе выхад?"
+                "trn": "Dze vyhad?",
+                "val": "Дзе выйсці?"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: I think the `trn` needs to change to match the new `val`

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 17]
```diff
@@ -23,224 +23,277 @@
...
             {
                 "key": "Can you help me?",
+                "trn": "Cі mozhace vy mne dapamagchy?",
                 "val": "Ці можаце вы мне дапамагчы?"
             },
             {
                 "key": "Where is the toilet?",
-                "val": "Дзе знаходзіца туалет?"
+                "trn": "Dze znahodzіca tualet?",
+                "val": "Дзе знаходзіца прыбіральня?"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: I think the `trn` needs to change to match the new `val`

**Comment by tagawa** [comment]
Thank you again @qqq3 and @mialeska. The latest version is on the beta server: https://beta.duckduckgo.com/?q=Belarusian%20Cheat%20Sheet
so if there are no further comments then we can merge this soon.

**Comment by tagawa** [comment]
No further comments so let's get merging.
@qqq3 - thank you for making these improvements to the cheat sheet, and thanks also @mialeska for the feedback. Дзякуй!

**Pull request merged**